plugins/chat/app/queries/chat/messages_query.rb

```diff
@@ -28,6 +28,7 @@ class MessagesQuery
     # @param thread_id [Integer] (optional) The thread ID to filter messages by.
     # @param target_message_id [Integer] (optional) The message ID to query around.
     #   It is assumed that the caller already checked if this exists.
+    # @param target_date [String] (optional) The date to query around.
```

Comment:
Does the date need to be in a particular format? If so we need to probably put this in the comment here / check for this in the query